src/components/TrapFocus/TrapFocus.tsx

```diff
@@ -68,25 +78,25 @@ export class TrapFocus extends React.PureComponent<TrapFocusProps, State> {
   };
 
   private handleBlur = (event: FocusEvent) => {
-    const {relatedTarget} = event;
+    const {relatedTarget: currentTarget} = event;
     const {focusTrapWrapper} = this;
     const {trapping = true} = this.props;
 
-    if (relatedTarget == null || trapping === false) {
+    if (trapping === false) {
```

Comment:
Was the extra check causing issues? `relatedTarget == null || `

Reply:
We'll probably want to keep the check since `relatedTarget` can be `null`. We can slightly change it though. `relatedTarget instanceOf HTMLElement` we are also able to remove the type cast

Reply:
In find in some cases, `relatedTarget` can be `null` if the next focusable node is outside of the page content, like the browser search bar. In these cases, I think we still want to manage focus and avoid the early `return`.

Reply:
@dleroux @AndrewMusgrave After checking with Devon, we do want to keep focus within the modal and prevent tabs from focusing browser elements (search bar, tabs). A user can always press the escape key, or navigate to the x button if they want to close the modal.

According to [this](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) ARIA example, 

> When focus is on the last focusable element in the dialog, moves focus to the first focusable element in the dialog.

You can also see this in action here: https://svinkle.github.io/launchy/

That means we should keep the change to remove the check `relatedTarget == null ` so that in the case that focus leaves the window content making it null, we don't have an early return (since our focus logic comes after this check).

Reply:
Still slightly worried about this, since `relatedTarget` has other situations where is can be null. We'll want a solid 🎩  in web when we 🚢 this.